### ISSUE TITLE
Not all reactions in the "golden" dataset pass your proposed chemical reaction standardization protocol

### ISSUE THREAD

**Issue by UnixJunkie** [Laboratoire-de-Chemoinformatique/Reaction_Data_Cleaning#5]
Apparently, only 1730 reactions pass your standardization protocol (scripts/standardizer.py);
while the dataset (data/golden_dataset.zip) contains 1851 reactions.

**Comment by ArkadiyLin** [comment]
Hello. We need some time to check that. We will return to you ASAP.

вт, 15 мар. 2022 г., 05:21 Francois Berenger ***@***.***>:

**Comment by UnixJunkie** [comment]
The ones which passed standardization on my computer are attached.
[golden_dataset_std.rdf.gz](https://github.com/Laboratoire-de-Chemoinformatique/Reaction_Data_Cleaning/files/8250347/golden_dataset_std.rdf.gz)

**Comment by ArkadiyLin** [comment]
I have reinstalled the packages required by our scipt with the versions mentioned on the README page. I have reobtained the results you have reported. As I suppose, you run a command like:

python3.7 standardizer.py -i ../data/golden_dataset.rdf -o ../data/golden_dataset_out.rdf -id Reaction_ID --logFile ../data/golden_dataset.log --skip_tautomerize


Indeed, this command returns 1730 reactions. However, this is not the full command line we used. As it is reported in our paper, we recomend to keep reactions even if reagents or products do not contain counterions (e.g., a salt like CCC[NH+].[Cl-] is mentioned only as CCC[NH+]). Thus, adding the option --keep_unbalanced_ions  returns you 1849 reactions. Unfortunately, 2 reactions were discarded by ValenceChecking. Latter can be explained by the inability to install CGRTools of exactly 4.0.36 version (only a bit earlier 4.0 version is available now). Nevertheless, we think that the protocol and the results are reproducible.

To help new users to obtain the same results as we did, we add the recomended command line to the README page.
